app/src/lib/stores/app-store.ts

```diff
@@ -1769,6 +1777,12 @@ export class AppStore extends TypedBaseStore<IAppState> {
       askForConfirmationOnForcePushDefault
     )
 
+    const strategy = localStorage.getItem(
+      uncommittedChangesStrategyKindKey
+    ) as UncommittedChangesStrategyKind
```

Comment:
I'd really like to avoid using `as` here, though I'm not sure of a good alternative approach yet. `localStorage.getItem` can return `undefined` if the given key doesn't exist, and this implementation tells the compiler that 'undefined' is a valid `UncommittedChangesStrategyKind`, which will result in runtime errors.